### ISSUE TITLE
Inconsistent JSON Request & Response Params

### ISSUE THREAD

**Issue by subins2000** [varnamproject/varnamd#25]
Currently varnamd's JSON requests and responses are very inconsistent. Some has capitalized JSON properties, others full small letters. This inconsistency should be fixed.

Examples :
https://api.varnamproject.com/languages
https://api.varnamproject.com/tl/ml/Malayalam

Same problem with request params

**Comment by joicemjoseph** [comment]
https://api.varnamproject.com/tl/ml/malayalam and 
https://api.varnamproject.com/tl/ml/Malayalam
returns same response.

(It is because of language feature.)

**Comment by subins2000** [comment]
I mean the response JSON params. While https://api.varnamproject.com/tl/ml/Malayalam returns with JSON properties all small letters (`success`), https://api.varnamproject.com/languages reeturns with `DisplayName` like structure

**Comment by subins2000** [comment]
The standard JSON outputs usually have property names all small letters, IMO we should use full lowercases and underscores if needed.

[Sample Wikipedia JSON output](https://en.wikipedia.org/api/rest_v1/page/media-list/Mangad)

Their output also uses camelCase and underscore. They're too inconsistent, what shall we stick to ? My vote is for small letters and underscore

**Comment by joicemjoseph** [comment]
> The standard JSON outputs usually have property names all small letters, IMO we should use full lowercases and underscores if needed.
> 
> [Sample Wikipedia JSON output](https://en.wikipedia.org/api/rest_v1/page/media-list/Mangad)
> 
> Their output also uses camelCase and underscore. They're too inconsistent, what shall we stick to ? My vote is for small letters and underscore

But, if we are migrating to new editor, we can change.

**Comment by subins2000** [comment]
Yeah, where should the change be to is the question ? Small case & underscore is a better way IMO

**Comment by joicemjoseph** [comment]
This will be a breaking change. @navaneeth, what is your comment on this ? 
?

**Comment by subins2000** [comment]
Related: #8 

From that, I infer lowercase is preferred

**Comment by joicemjoseph** [comment]
We can change to lowercase.
This change could break some third party apps depending on varnam api. 
We can do list out them and send a notification regarding this breaking change.
Otherwise, this may make some issues for those third party apps.

**Comment by subins2000** [comment]
For packs, I have chosen lowercase. This issue remains. This is a good beginner issue to work on. So I'm gonna keep this as is.
